README.md

```diff
@@ -1,9 +1,31 @@
 # Direct Deposit Module for Maker
 ![Build Status](https://github.com/makerdao/dss-direct-deposit/actions/workflows/.github/workflows/tests.yaml/badge.svg?branch=master)
 
-The Direct Deposit Module interfaces with third party lending protocols to enable a maximum variable borrow rate for selected assets. Maker Governance is able to pick a target variable borrow rate, and the module will automatically deposit/remove liquidity to ensure that target rate is hit.
+The Dai Direct Deposit Module (D3M) is a tool for directly injecting DAI into third party protocols.
 
-## DssDirectDepositAaveDai
+![D3M](https://imgur.com/kiV7g2f.png)
```

Comment:
I like this image.  You might add somehow that the pool holds any relevant balances (i.e. vat debt and any protocol tokens)

Reply:
done